test/e2e/select.test.js

```diff
@@ -142,6 +142,49 @@ describe('vl-select', async () => {
     await assert.eventually.isTrue(select.hasValue('two'));
   });
 
+  it('als gebruiker kan ik een select dynamisch activeren met gegroepeerde opties', async () => {
+    const select = await vlSelectPage.getDynamicGroupingSelect();
+    await assert.eventually.isFalse(select.isDressed());
+    assert.equal((await select.groups()).length, 0);
+    
+    await vlSelectPage.activeerDynamischeDataGrouping();
+
+    const selectMetData = await vlSelectPage.getDynamicGroupingSelect();
+    await selectMetData.open();
+    await assert.eventually.isTrue(selectMetData.isDressed());
+    
+    const groups = await selectMetData.groups();
+    assert.equal(groups.length, 2);
+    await assert.eventually.equal(groups[0].label, 'Group 1');
+    await assert.equal(groups[0].options.length, 2);
+    await assert.eventually.equal(groups[0].options[0].label, 'Label one');
+    await assert.eventually.equal(groups[0].options[0].value, 'one');
+
+    await assert.eventually.equal(groups[1].label, 'Group 2');
+    assert.equal(groups[1].options.length, 2);
+
+    await selectMetData.selectByValue('one');
+    await assert.eventually.equal(selectMetData.getSelectedValue(), 'one');
+  });
+
+  it('als gebruiker kan ik een de groepen van een undressed select opvragen', async () => {
```

Comment:
Waarom moet dat kunnen bij een undressed select?

Reply:
omdat ik wou testen dat we vanuit vl-select eenzelfde API kunnen aanbieden, of ge nu met nen dressed of undressed select bezig zijt.

Reply:
Wist niet dat een vl-select ooit undressed zou kunnen zijn, dacht dat hij zichzelf zou dressen bij gebrek aan dressing door de gebruiker.